securetea/core.py

```diff
@@ -64,16 +70,26 @@ def __init__(self):
             cred['twitter'],
             cred['debug']
         )
+        self.telegram = SecureTeaTelegram(
+            cred['telegram'],
+            cred['debug']
+        )
         if not self.twitter.enabled:
             self.logger.log(
                 "Twitter not configured properly. Exiting...",
                 logtype="error"
             )
             sys.exit(0)
-
+        if not self.telegram.enabled:
+            self.logger.log(
+                "Telegram not configured properly. Exiting...",
+                logtype="error"
+            )
+            sys.exit(0)
```

Comment:
Don't force to configure a telegram. Make sure its not blocking the user.

Reply:
Sure.